tests/roots/test_autodoc/baseFunction.m

```diff
@@ -1,3 +1,8 @@
 function y = baseFunction(x)
 % Return the base of x
+%
+% See Also:
+%     target.submodule.ClassMeow
```

Comment:
Wouldn't the regular way to list this be `ClassMeow` and `package.ClassBar` to make it compatible with MATLAB?

Reply:
Yes, but you'd have to turn on `matlab_short_links`.